### PR TITLE
Update AMS to v270 & finalize docker integration

### LINKED ISSUE #2: Migrate to V2.7.0
Referring to issue https://github.com/ImencoAS/insight/issues/707, we need to migrate to version 2.7 of AMS. This means we will need to pull the latest changes from the fork, but still keep the changes we have done.

### PR DESCRIPTION
**Pull request by RubenLarsen**
* Sets up everything to pull from correct docker images
* Fixes some issues with docker integration, more specifically WebRTCAppEE application deployment and Imencofied web UI

### CODE REVIEW

File: WebRTCAppEE/play.html [hunk 1]
```diff
@@ -74,711 +48,53 @@
...
-				document.getElementById("remoteVideo").play();
-				document.getElementById("videoPausedOverlay").style.display = "none";
-			}
-		}, false);
 
-		window.addEventListener('load', function() {
-			//run main method after everything is loaded
-    		main();
-		});
-
```
Comment (Andreas-Froyland) [inline WebRTCAppEE/play.html:779]: This is needed for the play/pause to function.

File: WebRTCAppEE/play.html [hunk 2]
```diff
@@ -3,68 +3,42 @@
...
-	<!-- HLS Player -->
 	<div id="video_container">
-		<video id="video-player" class="video-js vjs-default-skin vjs-big-play-centered" controls="hidden" preload="auto">
-			<p class="vjs-no-js">
-				To view this video please enable JavaScript, and consider upgrading
-				to a web browser that <a href="http://videojs.com/html5-video-support/" target="_blank">supports
-					HTML5 video</a>
-			</p>
-		</video>
+		<video id='video-player' class='video-js vjs-default-skin vjs-big-play-centered' controls playsinline></video>
```
Comment (Andreas-Froyland) [inline WebRTCAppEE/play.html]: controls should be hidden

File: WebRTCAppEE/play.html [hunk 3]
```diff
@@ -3,68 +3,42 @@
...
-		<!-- <span style="z-index: 2; position: absolute; top: 70%; left: 50%; transform: translate(-50%, -50%); color: white;"> Video Paused - Right Click => "Play" to unpause</span> -->
-		<div style="opacity:50%; background-color: transparent; z-index: 1; position: absolute; top: 0; left: 0; overflow: hidden; height: 100% !important; width: 100% !important; "> </div>
-	</div>
-
-
-	<!-- 360 player is added dynamically -->
-	
-
-	<div id="networkWarning"></div>
+	<div id="networkWarning">Your connection isn't fast enough to play this stream!</div>
```
Comment (Andreas-Froyland) [inline WebRTCAppEE/play.html]: Looks like networkWarning is handled inside embedded-player, so should we remove this text?  Or is the warning appearing often no longer a problem?

File: WebRTCAppEE/play.html [hunk 4]
```diff
@@ -3,68 +3,42 @@
...
-	<script src="js/fetch.js"></script>
-	<script src="js/promise.min.js"></script>
-	<script src="js/external/adapter-latest.js"></script>
-
 	<link href="css/player.css" rel="stylesheet">
+
 </head>
 
-<body style="background-color: transparent;">
+<body>
```
Comment (Andreas-Froyland) [inline WebRTCAppEE/play.html]: Body needs to be transparent

File: WebRTCAppEE/play.html [hunk 5]
```diff
@@ -3,68 +3,42 @@
...
 
 	<div id='video-overlay'>
-		<!-- <img src="images/loading.gif" alt="loading image" /> -->
+		<img src="images/loading.gif" alt="loading image" />
 	</div>
 
-	<div id="video_info" style="display: none">
-		Stream will start playing automatically
+	<div id="video_info">
```
Comment (Andreas-Froyland) [inline WebRTCAppEE/play.html]: set to display:none

File: WebRTCAppEE/play.html [hunk 6]
```diff
@@ -3,68 +3,42 @@
...
-		style="object-fit: fill;">
-	</video>
-
-
-
-	<div id="videoPausedOverlay" style="display: none;" >
-		<img id="pauseIcon" src="images/icons/pause.svg" style="z-index: 2; width: 15%; position: absolute; top: 50%; left: 50%; transform: translate(-50%, -50%);"/>
-		<!-- <span style="z-index: 2; position: absolute; top: 70%; left: 50%; transform: translate(-50%, -50%); color: white;"> Video Paused - Right Click => "Play" to unpause</span> -->
-		<div style="opacity:50%; background-color: transparent; z-index: 1; position: absolute; top: 0; left: 0; overflow: hidden; height: 100% !important; width: 100% !important; "> </div>
-	</div>
```
Comment (Andreas-Froyland) [inline WebRTCAppEE/play.html]: This is needed for the pause icon to appear

**Comment by Andreas-Froyland** [comment]
@RubenLarsen could you try the new changes?

**Comment by Andreas-Froyland** [comment]
Well, I messed up something, causing the files I changed to not appear in the files changed in the PR.

If you pull the latest code from the branch, you can see that there is a new folder called imenco-js. This has a copy of the `embedded-player.js` but with some changes to enable pausing etc...

**Comment by RubenLarsen** [comment]
A full IMS deployment can be tested by following this guide: https://wiki.imenco-dev.com/en/projects/IMS

This should confirm this PR(because antmediaserver-base:2.7.0 has been pushed to docker hub) as well as confirm that the imenco-specific things are deployed automatically.

**Comment by Andreas-Froyland** [comment]
Tested and works great:)

**Pull request merged**